packages/main/src/Icon.ts

```diff
@@ -368,6 +368,10 @@ class Icon extends UI5Element {
 			return console.warn("Icon name property is required", this);
 		}
 
+		await this.loadIconData(name);
+	}
+
+	async loadIconData(name: string) {
```

Comment:
`loadIconData` is basically the entire Icon's onBeforeRendering. While for the Icon, it's just refactoring, it's meant for usage by the Button's onAfterRendering. This is more or less calling the Icon's onBeforeRendering in the Button's onAfterRendering.
And, it will be called once more normally with the Icons rendering lifecycle;

I guess the problem is that the Icon rendering is async and Button's not, that's why you can't just access "effectiveAccessibleName" on the Icon instance during Button's rendering.

At least, we can extract the minimum logic in a method like this to get the data the Button needs:
```js
async getAccessibleName () => {
    if (this.accessibleName) {
       return this.accessibleName;
    }
  
    let iconData = getIconDataSync(name);
    
    if (!iconData) {
	    iconData = await getIconData(name);
	}

    if (iconData && iconData.accData) {
        const i18nBundle = await getI18nBundle(iconData.packageName);
		return i18nBundle.getText(iconData.accData) || undefined;
    }
}
```

@vladitasev what do you think